server/scripts/emails.ts

```diff
@@ -41,8 +47,83 @@ export const createNotificationEmailObject = (notification_data: IPostNotificati
     to: null,
     from: 'Commonwealth <no-reply@commonwealth.im>',
     subject: subjectLine,
+    templateId: 'd-7ccab17c309f45dab93b4290b3d396d1',
```

Comment:
should these be private in any way?

Reply:
I don't think so. It's just a hash that sendgrid uses to format the email prior to sending. Our sendgrid API key is already in our environment variables and that's what triggers the email sending anyways. The hash is created at the time in which you user their WYSIWYG dynamic email template creator. It's how they know which of the template formats to use.